xero_files.yaml

```diff
@@ -518,6 +518,42 @@ paths:
                               "ObjectGroup":"Contact"
                           }
                         ]'
+  /Associations/Count:
+    parameters:
+      - $ref: '#/components/parameters/requiredHeader'
+    get:
+      security:
+        - OAuth2: [files, files.read]
+      tags:
+        - Files
+      summary: Retrieves a count of associations for a list of objects.
+      operationId: getAssociationsCount
+      description: By passing in the appropriate options, 
```

Comment:
Looks like the description is incomplete?

Reply:
Updated. I also updated in another place where it was also missing.